libs/utils/energy.py

```diff
@@ -134,41 +142,33 @@ def __init__(self, target, hwmon_conf=None, res_dir=None):
         self.readings = {}
 
         if 'hwmon' not in self._target.modules:
-            logging.info('%14s - HWMON module not enabled',
-                    'EnergyMeter')
+            logging.info('%14s - HWMON module not enabled', 'HWMon')
             logging.warning('%14s - Energy sampling disabled by configuration',
-                    'EnergyMeter')
+                            'HWMon')
```

Comment:
I would suggest to keep these cleanup/renaming chunks in a separate "cosmetics patch"... maybe at the beginning of the series